src/app/components/Loop/QuoteModal.tsx

```diff
@@ -0,0 +1,264 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Modal, message } from 'antd';
+import { AppState } from 'store/reducers';
+import { getLoopOutQuote, getLoopOut, getLoopIn } from 'modules/loop/actions';
+import { Button, Icon, Alert } from 'antd';
+import { ButtonProps } from 'antd/lib/button';
+import { GetLoopOutArguments, GetLoopInArguments } from 'lib/loop-http/types';
+
+interface StateProps {
+  loopQuote: AppState['loop']['loopQuote'];
+  loop: AppState['loop']['loop'];
+  error: AppState['loop']['error'];
+  hasPassword: boolean;
+}
+
+interface DispatchProps {
+  getLoopOutQuote: typeof getLoopOutQuote;
+  getLoopOut: typeof getLoopOut;
+  getLoopIn: typeof getLoopIn;
+}
+
+interface OwnProps {
+  amt: string;
+  sct: string;
+  isOpen?: boolean;
+  type: string;
+  dest: string;
+  srf: string;
+  sf: string;
+  mf: string;
+  pre: string;
+  chan: string;
+  adv: boolean;
```

Comment:
Many of these props are hard to understand without context, I would keep them to their full names for better readability (even though it's somewhat more verbose.)

Reply:
I just got lazy there! My bad. Made it more verbose!